framework/build.sh

```diff
@@ -7,25 +7,27 @@ cd build
 
 cmake \
   -G Ninja \
-  -DUSE_SYSTEM_EIGEN=ON \
-  -DUSE_JEMALLOC=OFF \
-  -DENABLE_OPENGL=OFF \
-  -DENABLE_DOCS=OFF \
-  -DENABLE_MANTIDPLOT=FALSE \
-  -DENABLE_WORKBENCH=FALSE \
-  -DENABLE_OPENCASCADE=FALSE \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DUSE_JEMALLOC=ON \
```

Comment:
I'm not sure there is much use in enabling this. We only use it in the main package on Linux through `LD_PRELOAD`, which requires using the `mantidpython` wrapper scripts.